More comments and tests for `FilenameToStream`

Calvin-L opened the pull request:
This commit also normalizes the whitespace in `FilenameToStream` to four spaces (instead of a mix of tabs and spaces).

lemmy commented:
LGTM

By the way, I don't remember exactly but TLAFile could become obsolete with the (unscheduled) demise of the Toolbox.

Calvin-L commented:
Whoops! Looks like this broke the toolbox build because I moved one static method and removed another:
https://github.com/tlaplus/tlaplus/actions/runs/6726425505/job/18282651608

I'll have a fix ready shortly.

Calvin-L commented:
Fixed in b40b2f3b1003800741c9726a84188f7eae164ad2 and 7ef5199eb0fbad8541da7733bcac75c6f7647eb1.

lemmy commented:
Related: https://github.com/tlaplus/tlaplus/issues/719
